Persistence/Entities/Courier.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace shipping_service.Persistence.Entities
+{
+    public class Courier
+    {
+        public ulong Id { get; set; }
+        public string Username { get; set; }
+        public DateTime Created { get; set; }
+        public DateTime Modified { get; set; }
+        public byte[] HashedPassword { get; set; }
+        public string Name { get; set; }
```

Comment:
Not sure why we need the name?

Reply:
I just thought since the courier is empoyed by us or something, we should know his name for this to be more official. We can remove this as we definitely won't have time to use this meaningfully in the application.

Reply:
When we would need his surname and code probably?